js/scriptClients.js

```diff
@@ -0,0 +1,123 @@
+
+const clientsContainer = document.querySelector('.clients-container');
+const showOpinionsBtn = document.querySelector('.show-opinions-btn');
+const ratings =[];
+const createHighRatings = index =>{
+    for(let i =80; i<=100; i++ ){
+        ratings.push(i);
+    }
+    return ratings[index];
+}
```

Comment:
proponuję uproszczenie całej zabawy z generowaniem fejkowych ocen:

może lepiej użyć jakiejś funkcji jak w tym linku:
https://stackoverflow.com/questions/1527803/generating-random-whole-numbers-in-javascript-in-a-specific-range
![image](https://user-images.githubusercontent.com/98218505/153721394-a18895dd-c48a-47c9-8f19-1d854335e620.png)

![image](https://user-images.githubusercontent.com/98218505/153721440-16e77bd1-cf4b-4739-a023-aff3a0fb8c91.png)

moja propozycja:
![image](https://user-images.githubusercontent.com/98218505/153721516-8a573909-a8f1-4065-892d-d8905b01390c.png)